lib/services/post_service.dart

```diff
@@ -69,7 +69,7 @@ class PostService {
   ///   None
   ///
   /// **returns**:
-  /// * `Future<void>`: define_the_return
+  /// * `Future<void>`: returns future void
```

Comment:
You can write None for Future<void> too. I guess ? Please let me know if that's not the case and I'll update the linter.

Reply:
Yes, just like how we are showing nothing for void function we do the same for future void too. If you won't mind sir, may I fix this? This way I will get to learn about custom linting practices and all.